### ISSUE TITLE
Vite can't resolve marked when using pnpm on sveltekit

### ISSUE THREAD

**Issue by ceifa** [pablo-abc/svelte-markdown#27]
I'm converting my project to use pnpm instead of regular npm, but when I tried to run the project I received the following error:

> Failed to resolve force included dependency: svelte-markdown > marked

And I fixed it adding this property to `svelte.config.js` (a workaround):

```json
{
    "vite": {
        "optimizeDeps": {
            "exclude": ["marked"]
        }
    }
}
```

I have a lot of dependencies and svelte-markdown was the only one with this problem, so I'm guessing it is something that should be fixed here, but I really don't know.

**Comment by FunMiles** [comment]
I hit the exact same problem. At least I am glad to see there's a workaround.

**Comment by Raicuparta** [comment]
#32 should solve this by updating marked

**Comment by ceifa** [comment]
Fixed in 0.2.0
